_posts/2020-07-29-PC-Engines-Firmware-v4-12-0-3.md

```diff
@@ -0,0 +1,213 @@
+---
+layout: post
+title:  "PC Engines apu coreboot Open Source Firmware v4.12.0.3"
+date:   2020-07-31
+categories: Firmware
+---
+# PC Engines apu coreboot Open Source Firmware v4.12.0.3
+
+## Key changes
+
+Mainline:
+
+1. **Rebased** with official coreboot repository **commit** 8d5cedf.
+2. **Fixed** memory speed values in the DMI table. Previous values were the
+speed of the bus not the memory thus it was 2 times smaller.
+3. **Fixed** cbmem error caused by wrong ACPI table.
+4. **Updated** sortbootorder to version 4.6.19 with an option in the runtime
+configuration allowing to reverse PCI addressing order. Previously, if a new PCI
+device was added to the PCIe slot, all other devices' addresses were incremented,
+which could result in renaming the internet interfaces.
+
+Legacy:
+1. **Updated** sortbootorder to version 4.6.19 with an option in the runtime
+configuration allowing to reverse PCI addressing order. Previously, if a new PCI
+device was added to the PCIe slot, all other devices' addresses were incremented,
+which could result in renaming the internet interfaces.
+
+## coreboot community
+
+Patches merged by community:
+
+* [mb/protectli/vault_kbl: Enable Intel PTT](https://review.coreboot.org/c/coreboot/+/42565)
+
+
+**Total:**
+
+* 6 lines added
+
+in official coreboot repository.
+
+## Statistics
+
+![Files Changed](https://cloud.3mdeb.com/index.php/s/JRiyzzy2se8mLaz/preview)
+
+The chart shows the total files changed from release tag against the rebase
+point of given release specified in CHANGELOG (CHANGELOG.md and gitlab-ci.yml
+excluded from statistics). Check the statistics with:
+
+```
+git diff --stat 8d5cedf ':(exclude).gitlab-ci.yml' ':(exclude)CHANGELOG.md'
```

Comment:
I saw that we have added another script for CI, so the exclude should be extended with ` ':(exclude).gitlab-ci/regression.sh'`